client/components/jetpack-header/style.scss

```diff
@@ -1,3 +1,7 @@
 .jetpack-header {
 	text-align: center;
 }
+
+.jetpack-header > svg {
```

Comment:
How do you feel about nesting this selector, so it would become:

```
.jetpack-header {
	text-align: center;

	& > svg {
		max-width: 100%;
	}
}
```